packages/extension/package.json

```diff
@@ -21,6 +21,7 @@
     "@dailydotdev/shared": "*",
     "classnames": "^2.3.1",
     "date-fns": "^2.25.0",
+    "date-fns-tz": "1.0.0",
```

Comment:
Do we need both?

Reply:
Yes the tz is a "plugin" on top of date-fns related to do all the timezone query stuff.
I've chosen this one so we didn't need to load a complete other polyfill.